go-controller/pkg/ovn/master.go

```diff
@@ -923,8 +923,8 @@ func (oc *Controller) ensureNodeLogicalNetwork(node *kapi.Node, hostSubnets []*n
 		// Create the Node's Logical Switch and set it's subnet
 		opModels = []libovsdbops.OperationModel{
 			{
-				Model:          &logicalSwitch,
-				ModelPredicate: func(lr *nbdb.LogicalSwitch) bool { return lr.Name == nodeName },
+				Model: &logicalSwitch,
+				//ModelPredicate: func(lr *nbdb.LogicalSwitch) bool { return lr.Name == nodeName },
```

Comment:
@JacobTanenbaum this looks like left-over debugging? I think we need the predicate bit, don't we?

Reply:
I removed the comment, we only need `ModelPredicate` if there may be more then one applicable switch. Since the model contains an index into the `logicalSwitch` table, the UUID, there is only one object that this can apply too. Done